microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -3,19 +3,20 @@ description: REST endpoints to automate publishing updates to add-ons that are s
 title: Using the Microsoft Edge Add-ons API
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 08/10/2021
+ms.date: 08/19/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, add-ons api, publish api
 ---
 # Using the Microsoft Edge Add-ons API
 
+> [!NOTE]
+> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
```

Comment:
merge the feedback link into the Note?  (on both pages)
Merge these two, temporary intro/setup paragraphs?